episodes/04-expertise.md

```diff
@@ -247,6 +247,7 @@ moving on, the more this implication is magnified.
 Instead, consider asking "What questions do you have?" and leaving a healthy pause for consideration.
 This firmly establishes an expectation that
 people will, indeed, have questions, and should challenge themselves to formulate them.
+By additionally sign-posting upcoming Q&A periods, learners can mentally prepare and are less put on the spot.
```

Comment:
```suggestion
If you plan to ask for questions at specific points, you can further encourage learners to prepare questions by including checkpoints in a posted schedule or announcing them in advance. 
```

Reply:
I wonder if this is a more complete explanation of the suggested idea? I do worry a bit that people will interpret this as a suggestion to *restrict* questions to Q&A periods, but I'm not sure how to avoid that without making this unnecessarily verbose.

Reply:
I am not even sure that it is enough. If you announce that there will be a time for questions, you have to explicitly state that questions can be asked at all times as well (if that is what you want), otherwise I would feel the opposite is implied.